FirebaseTestProject/Views/NewPostForm.swift

```diff
@@ -10,31 +10,73 @@ import SwiftUI
 struct NewPostForm: View {
     @State private var title = ""
     @State private var postContent = ""
+    @State private var imageSourceType: ImagePickerView.SourceType?
+    @State private var image: UIImage?
+    
     @FocusState private var showingKeyboard: Bool
+    
     @StateObject private var submitTask = TaskViewModel()
+    
     @Environment(\.user) private var user
+
     
     var body: some View {
         Form {
             TextField("Title", text: $title)
                 .focused($showingKeyboard)
-            TextField("Post content", text: $postContent)
+            TextEditor(text: $postContent)
                 .focused($showingKeyboard)
+                .multilineTextAlignment(/*@START_MENU_TOKEN@*/.leading/*@END_MENU_TOKEN@*/)
+                .frame(width: 300, height: 300, alignment: .topLeading)
+                
+            if let image = image {
+                Image(uiImage: image)
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 200, height: 200, alignment: .center)
+            } else {
+                Image(systemName: "photo")
+                    .resizable()
```

Comment:
Can these attributes be consolidated?  Would be nice to not need to write out the same ones twice.

Reply:
Done!